pkg/build/controller/build/defaults/defaults_test.go

```diff
@@ -21,8 +22,13 @@ func TestProxyDefaults(t *testing.T) {
 		GitHTTPSProxy: "https",
 		GitNoProxy:    "no",
 	}
+	defaultsProxy := &configv1.ProxySpec{
+		HTTPProxy:  "http",
+		HTTPSProxy: "https",
+		NoProxy:    "no",
+	}
 
-	admitter := BuildDefaults{defaultsConfig}
+	admitter := BuildDefaults{defaultsConfig, defaultsProxy}
```

Comment:
how about adding a test that confirms the defaultsProxy values are being applied to the build pod containers?

Reply:
updated in current push